src/main/java/hudson/plugins/xvnc/Xvnc.java

```diff
@@ -92,7 +95,7 @@ public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher
 
         String cmd = Util.nullify(DESCRIPTOR.xvnc);
         if (cmd == null) {
-            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp";
+            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp " + (this.additionalArgs != null ? this.additionalArgs : "");
```

Comment:
Note this will have no effect when there is global command (`DESCRIPTOR.xvnc`) specified